linode/manager
# feat: [M3-7209] – VPC BETA Feedback button

## Description 📝
Add a VPC BETA Feedback button to the VPC landing and detail pages.

## Changes  🔄
- Optional `icon` prop added to `DocsLink.tsx` to make it reusable for this use case (instead of creating a separate component)
- BETA Feedback link added to VPC landing & detail pages

## Preview 📷
<details>
<summary>VPC Landing page</summary>

![Screenshot 2023-11-06 at 4 22 42 PM](https://github.com/linode/manager/assets/114682940/060d308d-097a-432b-a548-e47616b903bd)

![Screenshot 2023-11-06 at 4 22 50 PM](https://github.com/linode/manager/assets/114682940/86a0d661-c431-4f29-9e2b-b288ffbd4b04)
</details>

<details>
<summary>VPC Details page</summary>

![Screenshot 2023-11-06 at 4 23 05 PM](https://github.com/linode/manager/assets/114682940/54526b6e-ed51-4dc3-90c7-ec659fddf436)

![Screenshot 2023-11-06 at 4 23 11 PM](https://github.com/linode/manager/assets/114682940/6a31292b-733d-496d-9579-0d48425e3b4e)
</details>

## How to test 🧪

### Prerequisites
- An account with the proper VPC tags

### Verification steps
Go to the VPC landing and detail pages and confirm the `BETA Feedback` button displays with the appropriate icon and links to the feedback form.

Confirm that the normal Docs links throughout the app have not been adversely impacted.

## As an Author I have considered 🤔

*Check all that apply*
